sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/storage/queue/AzureStorageQueueAutoConfigurationTest.java

```diff
@@ -55,7 +57,34 @@ void accountNameSetShouldConfigure() {
                 assertThat(context).hasSingleBean(QueueServiceClientBuilderFactory.class);
             });
     }
-    
+
+    @Test
+    void queueNameSetShouldConfigureQueueClient() {
+        this.contextRunner
+            .withPropertyValues(
+                "spring.cloud.azure.storage.queue.account-name=sa",
```

Comment:
nit: How about using a more readable name instead of `sa`? For example: `account-name-1`